pastefile/app.py

```diff
@@ -76,6 +76,9 @@ def init_check_directories(_app):
 except KeyError:
     pass
 
+LOG.warning("===== Running config =====")
+for c,v in app.config.iteritems():
+  LOG.warning("%s: %s" % (c,v))
```

Comment:
It's a data you might want to validate a startup. It's displayed only once.

It's inspired by a lot of apps who display the statup config not only in debug mode. But also when you run it on prod. It will let you a trace of how the app was started in the log. Can be a good thing to debug afterward a prod issue